numba/tests/test_operators.py

```diff
@@ -1022,6 +1022,32 @@ def test_bitwise_not(self, flags=force_pyobj_flags):
     def test_bitwise_not_npm(self):
         self.test_bitwise_not(flags=Noflags)
 
+    def test_bitwise_float(self):
+        """
+        Make sure that bitwise float operations are not allowed
+        """
+        def assert_reject_compile(pyfunc, argtypes):
+            msg = 'expecting TypingError with compiling {}'.format(pyfunc)
+            with self.assertRaises(errors.TypingError, msg=msg):
+                compile_isolated(pyfunc, argtypes)
```

Comment:
Could the exception message be checked to assert it contains `invalid usage of <op>`?